addons/csi-vmware-cloud-director/vcloud-csi-config.yaml

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: vcloud-csi-configmap
+  namespace: kube-system
+data:
+  vcloud-csi-config.yaml: |+
+    vcd:
+      host: {{ required "Please provide VCD_URL" .Credentials.VCD_URL }}
+      org: {{ required "Please provide VCD_ORG" .Credentials.VCD_ORG }}
+      vdc: {{ required "Please provide VCD_VDC" .Credentials.VCD_VDC }}
+      vAppName: {{ default .Config.Name .Params.vApp }}
```

Comment:
@kron4eg so in our sample Terraform configs `vappName = cluster_name`. But hard tying it just to that value doesn't seem correct. Do you think just having a param is sufficient? Or should we introduce a field in https://github.com/kubermatic/kubeone/blob/master/pkg/apis/kubeone/v1beta2/types.go#L287

Reply:
Idk, seems OK like you've wrote it.